gammapy/modeling/models/core.py

```diff
@@ -20,6 +20,28 @@
 log = logging.getLogger(__name__)
 
 
+def _recursive_dict_filename_update(d, path):
```

Comment:
I am wondering whether these 2 functions should not be into the utils directory...